site/config.toml

```diff
@@ -53,3 +53,9 @@ baseName = "_redirects"
 
 [outputs]
 home = ["HTML", "REDIRECTS"]
+
+# privacy settings
+[privacy]
+  [privacy.youtube]
+    # enable the cookie-less youtube in built-in hugo shortcode
+    privacyEnhanced = true
```

Comment:
this ensures we get the same cookieless content as before, the new HTML produced by the shortcode is like:
```html
<div class="video-wrapper">
  <iframe src="//www.youtube-nocookie.com/embed/8KtmevMFfxA" allowfullscreen="" title="YouTube Video"></iframe>
</div>
```